tests/setupTests.ts

```diff
@@ -1,15 +1,19 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
-// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
 configure({ adapter: new Adapter() });
 
 window.matchMedia =
   window.matchMedia ||
   function () {
     return {
       matches: false,
+      dispatchEvent: () => false,
```

Comment:
Fleshed out these properties to remove the TypeScript error, and I guess related changes made the `eslint` stuff not matter anymore.